### PR TITLE
Added rules for TypeScript and Jest.

### PR DESCRIPTION
**Pull request by rskrz**

### CODE REVIEW

**Comment by ghost** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Looks like you edited a package.json file. Sometimes that indicates that you changed the dependencies of the project. See [this doc](https://github.com/foundersclubsoftware/eslint-shareable-web/blob/master/docs/versioning.md) to learn more.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 5dea644f8a096fd0f702a5c7fdb8d3634c23588f
</p>

File: example/package.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
   "license": "MIT",
   "dependencies": {
     "core-js": "^3.6.4",
+    "eslint-plugin-jest": "^23.8.2",
+    "jest": "^25.3.0",
```
Comment (levibostian) [inline example/package.json]: These should be devDependencies. Not dependencies. They are only used for development and not needed at runtime.

File: example/src/index.ts [hunk 2]
```diff
@@ -1,2 +1,4 @@
 const foo: any = "";
 console.log(foo);
+const CamelCased = "abc";
```
Comment (levibostian) [inline example/src/index.ts]: Technically camel cased is `camelCased` and not `CamelCased`.

File: index.js [hunk 3]
```diff
@@ -1,11 +1,15 @@
 module.exports = {
-  extends: [
-    "eslint:recommended",
-    "plugin:@typescript-eslint/recommended"
-  ],
+  "plugins": ["jest"],
+  "extends": ["plugin:@typescript-eslint/recommended", "plugin:jest/recommended"],
```
Comment (levibostian) [inline index.js]: Why was `eslint:recommended` removed?

**Review by levibostian — CHANGES_REQUESTED** [review]

File: index.js (hunk 3 above)
Comment (rskrz) [reply index.js]: Oops, this might have been a mistake when I added jest.

File: example/src/index.ts (hunk 2 above)
Comment (rskrz) [reply example/src/index.ts]: Fixed

File: example/package.json (hunk 1 above)
Comment (rskrz) [reply example/package.json]: Fixed

**Review by levibostian — DISMISSED** [review]

**Comment by levibostian** [comment]
@rskrz looks good, but your git commit message is too long. See [error in the CI server](https://travis-ci.com/github/foundersclubsoftware/eslint-shareable-web/jobs/321815665#L203). Do another git rebase with a newly edited commit message for that commit and push it up to fix this.

**Comment by rskrz** [comment]
Same PR but with shorter commit message

**Review by levibostian — APPROVED** [review]

**Comment by levibostian** [comment]
@rskrz Congrats! Your PR got merged! Pretty cool. 

Pro tip: In the future, a better commit message for: 
`fix: fixed Jest plugin, eslint plugin, camelCase` would be `refactor: fixed Jest plugin, eslint plugin, camelCase`. 

This is because that commit technically was not a bug fix that you did. It was you doing some misc work to satisfy the pull request review. Misc work to get the PR merged.

**Pull request merged**